datasource/index.js

```diff
@@ -198,10 +198,13 @@ module.exports = yeoman.generators.Base.extend({
 
   installConnector: function() {
     var connector = this.availableConnectors[this.connector];
-    var pkg = connector.package;
-    if (!pkg) return;
-
-    var npmModule = pkg.name;
+    var pkg = {};
+    if (connector) {
+      pkg = connector.package;
+      if (!pkg) return;
+    }
+    
+    var npmModule = pkg.name || connector || this.connector;
```

Comment:
@bajtos When I check the code again, I think it should be 
`var npmModule = pkg.name || this.connector` instead of 
`var npmModule = pkg.name || connector || this.connector`

The `connector` is either undefined or an object, not a string. Thought?